dev-guide/cluster-fleet-evaluation.md

```diff
@@ -0,0 +1,160 @@
+the telemetry data](#interacting-with-telemetry) to figure out the prevalence of
+the scenario that the change will introduce. Allow sufficient time (TBD) for
+customers to upgrade to the Z stream release. The upgrade data is also available
+in telemetry.
+
+At this point the team needs to validate that the telemetry data matches their
+expectations and make a decision if they still plan to move forward with the
+change. In particular the team needs to measure the level of impact the change
+has. If the percentage of impact is very low then it might be preferable to
+contact the affected customers. Alternatively if the percentage of impact is
+very high, it would be preferable that the team re-evaluate the change and
+investigate if there is a way to automate the fix process in a seamless manner.
+
+If the component team decides to move forward, they will then be responsible for
+writing a [Knowledge Centered Support](https://source.redhat.com/groups/public/cee-kcs-program/cee_kcs_program_wiki/kcs_solutions_content_standard_v20)
+(KCS) document providing sufficient information describing the scenario and how
+to resolve it. Refer to the [KCS section](#writing-a-kcs-document) for more
+information.
+
+Once the KCS article/solution is ready and the list of customers are available,
+the component team can alert the affected customers using the methods described
+in the [Alerting customers](#writing-a-kcs-document) reference section. Teams
+should make every attempt to use all methods of alerting customers (this ensures
+contact with customers is made), but are free to drop options if they feel using
+that method may not reach the proper audience or may be too expensive.
+
+**Note**: Before the team goes down this path it will be useful to discuss the
+outcome and expectations with their respective OpenShift staff engineer, product
+manager and PLM contact. PLM contacts are listed under *column S* in the
+[OCP Team Member Tracking sheet](https://docs.google.com/spreadsheets/d/1M4C41fX2J1nBXhqPdtwd8UP4RAx98NA4ByIUv-0Z0Ds/edit#gid=1382138347).
+
+## Reference
+
+### Interacting with telemetry
+
+#### Pre-requisites
+
+To access the telemetry data please follow
+[these instructions](https://help.datahub.redhat.com/docs/interacting-with-telemetry-data).
+Once you have access, you will be able to perform queries against the data
+collected from customer clusters.
+
+#### Running queries
+
+You can query the telemetry data either using the
+[Thanos](https://telemeter-lts.datahub.redhat.com/graph) or
+[Grafana](https://telemeter-lts-dashboards.datahub.redhat.com) interfaces. For
+example, if you have introduced a new _ConditionStatus/Reason_ called _FooBar_,
+it can be queried using:
+
+```sql
+count by (_id, ebs_account) (cluster_operator_conditions{name="component-name",condition="ClusterFleetEvaluation",reason="FooBar"} * on(_id) group_left(ebs_account) max by(_id, ebs_account) (id_version_ebs_account_internal:cluster_subscribed{}))
```

Comment:
this risks `ebs_account` instability in `id_version_ebs_account_internal:cluster_subscribed`.  Which is not all that likely.  But still, you may want to guard against it by using `topk`, like:

```
count by (_id, ebs_account) (
  cluster_operator_conditions{name="component-name",condition="ClusterFleetEvaluation",reason="FooBar"}
  * on (_id) group_left (ebs_account)
  topk by (_id) (1, id_version_ebs_account_internal:cluster_subscribed)
)
```